### ISSUE TITLE
Ploty image.save.as can't complete display Chinese

### ISSUE THREAD

**Issue by zhonon** [plotly/plotly.py#1106]
when i use plotly func
    plotly.plotly.image.save_as(self.fig, filename=filepath + '.png')

there is problem，it cannot complete display ,not only  Traditional-Chinese  but also Simplified -Chinese，and Simplified -Chinese is  more serious,  i think the problem is on unicode,but i can't find where to change it . 

   plotly.plotly.image.save_as(self.fig, filename=filepath + '.png')
        self.fig['layout']['yaxis4']['title'] = '客戶數'
        self.fig['layout']['yaxis4']['titlefont'] = dict(size=11)
        self.fig['layout']['yaxis5']['title'] = '客戶勝率'
        self.fig['layout']['yaxis5']['titlefont'] = dict(size=11)

![promble](https://user-images.githubusercontent.com/26518173/44015070-d1da1aae-9f01-11e8-9cb9-4d33c1bae827.png)

**Comment by jonmmease** [comment]
Hi @zhonon , thanks for the report, I can see what you mean.

Here's the full example I used:

```python
import plotly.graph_objs as go
import plotly.plotly as ply
fig = go.Figure(data=[go.Scatter(y=[2, 1, 4])])
chinese_char_str = '客戶數'
fig.layout.yaxis.title = chinese_char_str
ply.image.save_as(fig, filename='chinese_char_test.png')
```

![plot](https://user-images.githubusercontent.com/15064365/44025120-3623cf7c-9ebd-11e8-9a3d-bfdf23daeeec.png)

The good news is that when I test this using an early prototype version of our coming [Orca integration for static image export](https://github.com/plotly/plotly.py/issues/1105) it works properly:

![chinese_char_test_orca](https://user-images.githubusercontent.com/15064365/44025311-c4f291a2-9ebd-11e8-9b65-3da98e02b75d.png)

You can learn more about the orca project at https://github.com/plotly/orca.  Orca will be well integrated into plotly.py for version 3.2, but in the meantime you can install and run it as a command line application (See instructions in the README linked above).

Orca has a lot of advantages of the `plotly.plotly.image.save_as` approach, so that's where we're going to focus our future image export efforts. In particular, its open source, free to use, works fully offline, and supports both raster (png. jpeg) and vector (svg, pdf, eps) output formats.

If you're interested in how this all comes together please check out the links above and let us know what you think.

**Comment by zhonon** [comment]
thinks for you Answer。
so,there is no others way to solve this ?
i just want use this image in Email.
and solve this as simple as possible.

**Comment by jonmmease** [comment]
Have you tried displaying the figure and then saving it from the save
button on the toolbar?
(I'm on my phone so I can't test it right now)

**Comment by zhonon** [comment]
i have try Method on
https://plot.ly/python/email-reports/
but i only can get this. so it's has no updata for a years
![image](https://user-images.githubusercontent.com/26518173/44043008-5c553f18-9f54-11e8-9f21-50aaa8fe2313.png)

**Comment by jonmmease** [comment]
Hi @zhonon ,

Here's what I mean about using the toolbar to save the image.  Display the plot using one of the `plot` or `iplot` methods. Here I'll use `plotly.plotly.plot`.

```python
import plotly.graph_objs as go
import plotly.plotly as ply
fig = go.Figure(data=[go.Scatter(y=[2, 1, 4])])
chinese_char_str = '客戶數'
fig.layout.yaxis.title = chinese_char_str
ply.plot(fig, filename='chinese_char_test.png', auto_open=True)
```

This will open the plot in a new browser tab.  Then click the camera button in the toolbar to download the plot as a .png.

Here's a screen capture of the process:

![saveimagetoolbar](https://user-images.githubusercontent.com/15064365/44087878-257af092-9f8f-11e8-8404-70f2a61f617a.gif)

Hope that helps!

**Comment by jonmmease** [comment]
This use-case is now covered by the orca-based static image export coming in 3.2.0. See https://github.com/plotly/plotly.py/pull/1120

**Comment by jonmmease** [comment]
Update: Programmatic static image export support has been released in 3.2.0: See https://medium.com/@plotlygraphs/plotly-py-end-of-summer-updates-5422c98b9058
